draftlogs/5854_fix.md

```diff
@@ -0,0 +1 @@
+- Skip `"hoverinfo": "none"` trace display for hover modes [[#5854](https://github.com/plotly/plotly.js/pull/5854)]
```

Comment:
Please add 
``` , with thanks to @Domino987 for the contribution! ```
on the second line.
Thank you!

Reply:
Hi this looks even better. Thanks for this.